packages/airnode-operation/src/config/evm-dev-config.json

```diff
@@ -17,7 +17,7 @@
             { "type": "bytes32", "name": "to", "value": "USD" },
             { "type": "bytes32", "name": "_type", "value": "int256" },
             { "type": "bytes32", "name": "_path", "value": "result" },
-            { "type": "bytes32", "name": "_times", "value": "1000000" }
+            { "type": "bytes32", "name": "_times", "value": "100000" }
```

Comment:
I edited this back to `100000` because it would end up changing more things inside Airnode. This was introduced when creating fixtures for Airkeeper in https://github.com/api3dao/airnode/commit/5798e606b070d188344c70b5edb5ad68ad1e16d4#diff-04917c5d42958ab67a1a84d91bbab5dee575960c2fc7751f86ed6283682ca5d4L67

Reply:
The thing is that I changed that value because of the number of decimals the mocked api is [currently](https://github.com/api3dao/airnode/blob/f717c568e08c7fe896f2973bb75223eeff32ef23/packages/airnode-operation/src/server.ts#L16) returning. I had to add that extra 0 to be able to get an integer representation of the full number so it doesn't get truncated. What things need to change in Airnode? Should we take out a decimal in server.ts instead?

Reply:
>  What things need to change in Airnode?

The template IDs are changed.

> Should we take out a decimal in server.ts instead?

Yeah, I'd rather do that.

Reply:
It might brake some tests but I'm ok with that 👍🏻